docker/docker-compose.yaml

```diff
@@ -7,12 +7,12 @@ networks:
 services:
 
 # Python
-  python:
-    image: 'oriolfilter/python-rolelive:first_try'
+  rolelive:
+    build:
+      context: ./dockerfiles
+      dockerfile: python-rolelive
```

Comment:
Dockerfiles are better called like that, Dockerfile. We will not have any other dockerfiles for this project than the one for the Rolelive image. Remove the ``dockerfiles`` folder and rename ``python-rolelive`` to ``Dockerfile``. This is way more clear for people to know where is the Dockerfile.